Grave and apostrophe on the 4th level can be provided via simpler
keystrokes.  They were historically provided as alternative quotes,
but these one are better alternatives now that Unicode is common.
Closes: #308196  Thanks Mark Sheppard

Submitted upstream: https://bugs.freedesktop.org/show_bug.cgi?id=8122

Index: xkb-data/symbols/latin
===================================================================
--- xkb-data.orig/symbols/latin
+++ xkb-data/symbols/latin
@@ -50,8 +50,8 @@
     key <AB01>	{ [         z,          Z, guillemotleft,        less ]	};
     key <AB02>	{ [         x,          X, guillemotright,    greater ]	};
     key <AB03>	{ [         c,          C,         cent,    copyright ]	};
-    key <AB04>	{ [         v,          V, leftdoublequotemark, grave ]	};
-    key <AB05>	{ [         b,          B, rightdoublequotemark, apostrophe ] };
+    key <AB04>	{ [         v,          V, leftdoublequotemark, leftsinglequotemark ]	};
+    key <AB05>	{ [         b,          B, rightdoublequotemark, rightsinglequotemark ] };
     key <AB06>	{ [         n,          N ]	};
     key <AB07>	{ [         m,          M,           mu,    masculine ]	};
     key <AB08>	{ [     comma,       less, horizconnector,   multiply ]	};
Index: xkb-data/symbols/us
===================================================================
--- xkb-data.orig/symbols/us
+++ xkb-data/symbols/us
@@ -157,8 +157,8 @@
   key <TLDE> { [ dead_grave, dead_tilde,    grave,	      asciitilde    ] };
   key <AE05> { [          5, percent,	    EuroSign			    ] };
   key <AE06> { [	  6, dead_circumflex, asciicircum,    asciicircum   ] };
-  key <AE09> { [	  9, parenleft,	    dead_breve,	      dead_breve    ] };
-  key <AE10> { [	  0, parenright,    dead_abovering,   dead_abovering] };
+  key <AE09> { [	  9, parenleft, leftsinglequotemark,  dead_breve ] };
+  key <AE10> { [	  0, parenright, rightsinglequotemark, dead_abovering ] };
   key <AE11> { [      minus, underscore,    dead_macron,      dead_belowdot ] };
   key <AE12> { [      equal, plus,	    dead_doubleacute, dead_horn	    ] };
 
